docs/user-flows.md

```diff
@@ -40,7 +40,7 @@ In DevTools, navigation is easy: ensure it's the selected mode and then click _A
 ```js
 import {writeFileSync} from 'fs';
 import puppeteer from 'puppeteer';
-import lighthouse from 'lighthouse/lighthouse-core/fraggle-rock/api.js';
```

Comment:
Should we merge the docs change into a staging branch like we did with https://github.com/GoogleChrome/lighthouse/pull/14195?

Reply:
no I don't want to do that for this

Reply:
Why not?

Reply:
Oh cuz links will be broken all over the place right